core/src/main/resources/jenkins/model/Jenkins/configure.jelly

```diff
@@ -29,6 +29,7 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 <l:layout permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%Configure System}" type="one-column">
   <st:include page="sidepanel.jelly" />
+  <!-- no need for breadcrumb here -->
```

Comment:
what does this add?
```suggestion
```

Reply:
I didn't reproduced the body of my original PR, I should.

Here is the relevant bit:

> I've also added <!-- no need for breadcrumb here --> as comment in views where adding a breadcrumb item wasn't necessary, let me know if it's preferable to omit them.

See also Wadeck comment about it: https://github.com/jenkinsci/jenkins/pull/7457#issuecomment-1330975869

Fine for me not adding it.

Reply:
to me this seems like a development comment it also seems inaccurate as there is a breadcrumb on this page:

![image](https://user-images.githubusercontent.com/21194782/206399975-17032f75-d267-4e61-8fda-e8d34213dcf0.png)

Reply:
I had initially a longer comment, WDYT of:
```suggestion
  <!-- no need for additional breadcrumb here because we're on a page with an existing "Configure" breadcrumb -->
```

Reply:
This still doesn't make sense to me because it's using the `f:breadcrumb-config-outline` which adds a breadcrumb to the page. I don't think a comment makes sense when you are using a tag that adds a breadcrumb, the docs on the tag itself explain more about what it does.

Reply:
Sorry I didn't noticed that tag before, thanks, I agree with you then.

```suggestion
```

Reply:
I've created a docs ticket: https://github.com/jenkinsci/design-library-plugin/issues/182